conda/recipes/libraft/meta.yaml

```diff
@@ -170,19 +167,10 @@ outputs:
         - sysroot_{{ target_platform }} {{ sysroot_version }}
```

Comment:
Unrelated comment: libcudf has a subpackage called libcudf-example. It does something similar to libraft-template. I think that the name “example” makes more sense than “template,” especially when template can be an overloaded word for header-only libraries (which feature C++ templates).